### PR TITLE
Update to latest util/lm

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?zinc=pull/336/head&deploy_to_bintray=false">this link.</A>

File: build.sbt [hunk 1]
```diff
@@ -294,8 +306,11 @@ lazy val compilerInterface = (project in internalPath / "compiler-interface")
 // Compiler-side interface to compiler that is compiled against the compiler being used either in advance or on the fly.
 //   Includes API and Analyzer phases that extract source API and relationships.
 lazy val compilerBridge: Project = (project in internalPath / "compiler-bridge")
-  .dependsOn(compilerInterface % "compile;test->test",
-             /*launchProj % "test->test",*/ zincApiInfo % "test->test")
+  .dependsOn(
+    compilerInterface % "compile;test->test",
+//  launchProj % "test->test",
```
Comment (Duhemm) [inline build.sbt]: I think that the time to remove this comment has come.
https://github.com/sbt/sbt-zero-thirteen/blob/0.13/build.sbt#L380

**Review by Duhemm — DISMISSED** [review]
scalafmt is not enforced on this repo? I have no problem with the style changes I'm just surprised because there's no change to the configuration.

**Comment by dwijnand** [comment]
The following format "flavours" are allowed, with the current scalafmt configuration:

```scala
function(argument1,
         argument2)

// OR

function(
    argument1,
    argument2)

// OR

function(
    argument1,
    argument2
)
```

To enforce the last style (as I understand it) you have to opt into `danglingParentheses = true` and `align.openParenCallSite = false`. I haven't gotten round to PR'ing that.

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?zinc=pull/336/head&deploy_to_bintray=false">this link.</A>

**Pull request merged**